doc/developer-guides/hld/atkbdc-virt-hld.rst

```diff
@@ -0,0 +1,64 @@
+.. _atkbdc_virt_hld:
+
+atkbdc emulation
```

Comment:
Is there a more explicit term we could use here instead of `atkbdc`, something like "AT Keyboard Controller"?